tbxforms/templates/tbx/field.html

```diff
@@ -25,7 +25,7 @@
         {% if field.label and not field|is_checkbox and form_show_labels %}
             {% if label_tag %}<{{ label_tag }} class="tbxforms-label-wrapper">{% endif %}
             <label for="{{ field.id_for_label }}" class="tbxforms-label{% if label_size %} {{ label_size }}{% endif %}">
-                {{ field.label|safe }}
```

Comment:
At least TBXFORMS_ALLOW_HTML_BUTTON seemed fairly unnecessary to me as well. Still, I guess it's usually only a small number of fields where html is needed, so perhaps it's better to handle those cases individually and remove all three of these options. It might be enough to give an example in the readme how to do that exactly.